accelerator/config.c

```diff
@@ -48,24 +50,65 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
 #ifdef DB_ENABLE
   db_client_service_t* const db_service = &core->db_service;
 #endif
+  char* strtol_p = NULL;
+  long int strtol_temp;
   switch (key) {
     // TA configuration
     case TA_HOST_CLI:
       ta_conf->host = value;
       break;
     case TA_PORT_CLI:
-      ta_conf->port = value;
+      strtol_temp = strtol(value, &strtol_p, 10);
+      if (strtol_p != value && errno != ERANGE && strtol_temp >= INT_MIN && strtol_temp <= INT_MAX) {
```

Comment:
What if you encounter converting problem?

Reply:
I added a logger to report the error, and TA will use default value in this situation